Dockerfile

```diff
@@ -8,4 +8,6 @@ RUN go build ./cmd/proxy
 
 FROM alpine:3.12
 COPY --from=builder /go/github.com/ExchangeUnion/xud-docker-api-poc/proxy /usr/local/bin/proxy
+COPY ./cmd/proxy/cert.sh .
+RUN apk add --update openssl
```

Comment:
I suppose the https://github.com/ExchangeUnion/xud-docker/blob/master/images/proxy/Dockerfile has to be changed accordingly @reliveyy ?

Reply:
@krrprr Yes

Reply:
Thanks! Created a [PR](https://github.com/ExchangeUnion/xud-docker/pull/762) for that @reliveyy